front/src/config/i18n/en.json

```diff
@@ -1760,6 +1828,41 @@
       "stop": "Stop",
       "record": "Record"
     },
+    "energy-sensor": {
+      "shortCategoryName": "Energy sensor",
+      "voltage": "Voltage",
+      "current": "Current",
+      "binary": "Switch",
+      "power": "Power",
+      "energy": "Energy",
+      "index": "Index"
+    },
+    "volume-sensor": {
+      "shortCategoryName": "Volume sensor",
+      "decimal": "Volume (decimal)",
+      "integer": "Volume (integer)"
+    },
+    "currency": {
+      "shortCategoryName": "Currency",
+      "cost": "Cost"
```

Comment:
Same here, should be decimal

"Trading" or "Money" as category name

and inside "Currency"